src/main/scala/sbtlicensereport/SbtLicenseReport.scala

```diff
@@ -71,15 +75,18 @@ object SbtLicenseReport extends AutoPlugin {
       // Here we use an empty partial function
       licenseReportNotes := PartialFunction.empty,
       licenseOverrides := PartialFunction.empty,
+      licenseDepExclusions := PartialFunction.empty,
```

Comment:
While this should probably sit within `globalSettings`, as you can see there are other settings which should also sit there so I was thinking of fixing this in one go in a future PR?

Reply:
Sounds good